WeakAuras/RegionTypes/DynamicGroup.lua

```diff
@@ -815,6 +816,52 @@ local growers = {
       end
     end, anchorOn
   end,
+  CURVE = function(data)
+    local oX, oY = 0, 0
+    local radius = data.radius or 0
+    local limit = data.useLimit and data.limit or math.huge
+    local sAngle = (data.rotation or 0) * math.pi / 180
+    local arc = (data.fullCircle and 360 or data.arcLength or 0) * math.pi / 180
+    if data.reverseFill then
+      sAngle = sAngle + arc
+    end
+    local anchorPerUnitFunc, anchorOn
+    if data.useAnchorPerUnit then
+      anchorPerUnitFunc, anchorOn = createAnchorPerUnitFunc(data)
+    end
+    return function(newPositions, activeRegions)
+      local frames = {}
+      if anchorPerUnitFunc then
+        anchorPerUnitFunc(frames, activeRegions)
+      else
+        frames[""] = activeRegions
+      end
+      for frame, regionDatas in pairs(frames) do
```

Comment:
This code seems identical to me with the CIRCLE one if constanFactor is "RADIUS".

Reply:
A lot of it is the same - my changes were based on one of the circle growers. But there are key differences, which I mentioned in the PR, even when circle has the constant factor set to radius.

Am I missing your point, or have I not clearly described how this grower is different?

Reply:
Well, your description does not match the code. The code is identical as far as I can tell.

Reply:
I feel like you have to have missed something. This PR contains exactly the code that's on my hard drive, which is also the code that the WoW client loads when I test. My description matches exactly what happens in the game, and the code in this PR for the circle types and the curve type is also different. The code differences are verifiable with whatever diff tool you prefer. I'm really not sure what's going on, because even 12 lines up from the line you commented on there's an `if` block that doesn't exist in the circle growers. I can see it while typing this reply, in the PR UI, without even using a diff tool. If it helps, here's a copy/paste of the two growers compared in a diff tool - https://www.diffchecker.com/BnB0Ba1l/.

Reply:
Great: 
So let's ignore reverseFill for now.
Then you renamed numVisible to maxVisible
If we assume constatnFactor is "RADIUS", in the circle grow r is assigned radius, same as your code.

So where's the difference?

Reply:
Ah okay the difference is #regionData vs #data.controlledChildren.

Okay, we could have saved a lot of time if you pointed out the only change you actually did.

Reply:
> Great: So let's ignore reverseFill for now. Then you renamed numVisible to maxVisible If we assume constatnFactor is "RADIUS", in the circle grow r is assigned radius, same as your code.
> 
> So where's the difference?

It is quite literally not the same. I don't understand why this isn't clear.

The circle growers operate on active auras only. With the `constantFactor` set to `radius` the spacing between auras changes depending on how many auras are active at any given time. Functionally this also makes it look like the group fills in an alternating pattern, sort of bouncing from side to side.

The grower in this PR has the spacing between auras determined by the total number of auras in the group. It always places newly active auras next to the ones that were already active without anything else moving.

I could try to say more, but at this point I think the simplest way to understand would be to try it out in-game, or if you prefer I can record a video for you. They are very obviously different in function, and the "rename" is also part of an actual functional change too, not a simple rename. How would you like to proceed?

Reply:
> Ah okay the difference is #regionData vs #data.controlledChildren.
> 
> Okay, we could have saved a lot of time if you pointed out the only change you actually did.

That isn't the only change. It's a little more complex than that. I don't understand why you're being so defiant and dismissive rather.

Reply:
Additionally, I did mention that specific part of the change in my PR in the section about potential discussion. It's point 3.

Reply:
Where are any other changes? The renaming does nothing functional as far as I can tell. Again point to the exact line where your renaming changes the behaviour. You need to be able to communicate clearly.

Now onto the meat of your PR. That's the wrong approach. Grow functions should not depend on the total number of child auras due to cloning auras.

The better approach would be a separate setting for the angle between auras.